src/structures/Guild.js

```diff
@@ -1070,7 +1070,7 @@ class Guild extends AnonymousGuild {
    * @example
    * // Edit the guild owner
    * guild.setOwner(guild.members.cache.first())
-   *  .then(updated => console.log(`Updated the guild owner to ${updated.owner.displayName}`))
+   *  .then(guild => console.log(`Updated the guild owner to ${guild.members.resolve(guild.ownerId).displayName}`))
```

Comment:
Alternatively:
```suggestion
   *  .then(guild => guild.fetchOwner())
   *  .then(owner => console.log(`Updated the guild owner to ${owner.displayName}`))
```

Keeps lines shorter and reduces the code complexity, as well as doesn't rely on the member cache to... be populated (remember we have cache limits now). Best case scenario, the `fetchOwner()` call won't do a HTTP request because it prioritizes cache.